hoomd/md/methods.py

```diff
@@ -913,15 +913,19 @@ def _attach(self):
         super()._attach()
 
 
-class berendsen(_Method):
+class Berendsen(_Method):
     R""" Applies the Berendsen thermostat.
 
```

Comment:
Should we warn that this does not sample the canonical distribution properly, @joaander?

Reply:
I think HOOMD generally assumes that users choosing a thermostat know what they're doing wrt to the physics at this level, right? Or are we trying to provide guidance on what thermostat/barostat/etc to use v3? That seems out of scope for library documentation, but I might have missed that decision.

Reply:
Agreed. The library documentation should state what it is and what the parameters are.